devices/common_patches/fsmc/fsmc_sramfix.yaml

```diff
@@ -0,0 +1,33 @@
+"FMC,FSMC":
+  BCR1:
+    _add:
+      WRAPMOD:
+        description: WRAPMOD
+        bitOffset: 10
+        bitWidth: 1
+  "BCR[2-4]":
+    _add:
+      CCLKEN:
+        description: CCLKEN
+        bitOffset: 20
+        bitWidth: 1
+  BCR?:
+    _add:
+      WFDIS:
```

Comment:
I don't think RM0090 has WFDIS at all, but it covers e.g. F429 which uses this patch file.